### DISCUSSION TITLE
Sentry error reporting

### DISCUSSION THREAD

**Discussion by doc-sheet** [seaweedfs/seaweedfs#5333] [Ideas]
Hi!
I would like to connect seaweed instance to sentry, could you add at least basic support for it?
https://docs.sentry.io/platforms/go/#install

I'm ready to make simple PR but I hope someone else know better how to cover all entry points.

**Comment by chrislusf** [comment]
not attempted before.

**Reply by doc-sheet** [reply]
let's try https://github.com/seaweedfs/seaweedfs/pull/5410

**Comment by doc-sheet** [comment]
Thank you, I'll test it when I can and provide some tweaks in later PR.
